arraySort.js

```diff
@@ -3,29 +3,37 @@ var oddNumbers = new Array();
 var characters = new Array();
 
 let numbers = {
-    evens: evenNumbers,
-    odds: oddNumbers,
-    chars: characters,
+    'evens': evenNumbers,
+    'odds': oddNumbers,
+    'chars': characters,
 }
 
```

Comment:
Move all the code above to the body of the function.
This will help you prevent unwanted variable access and weird bugs, for example, Call the `arraySort` function twice as shown below and see whether the second result is true
```js
arraySort([1, 2, "x"]); // logs { evens: [ 2 ], odds: [ 1 ], chars: [ 'x' ] } which is okay
arraySort([4, 5, "y"]); // logs { evens: [ 2, 4 ], odds: [ 1, 5 ], chars: [ 'x', 'y' ] } which is not okay, but putting all the above variables in the function body fixes the problem
```